pythnet/message_buffer/programs/mock-cpi-caller/tests/cases/test_resize_buffer.rs

```diff
@@ -125,30 +125,47 @@ async fn fail_resize_buffer_invalid_increase() {
     let err: ProgramError = res.unwrap_err().into();
     assert_eq!(
         err,
-        ProgramError::Custom(MessageBufferError::TargetSizeDeltaExceeded.into())
+        ProgramError::Custom(anchor_lang::error::ErrorCode::AccountReallocExceedsLimit.into())
     );
+    println!("finished invalid buffer increase");
 
 
+    let msg_buffer_account_data = context
+        .fetch_msg_buffer_account_data(&msg_buffer_pda)
+        .await
+        .unwrap();
+
+    println!("MSG_BUFFER_ACCOUNT_DATA: {:X?}", msg_buffer_account_data);
+
+
+    assert_eq!(
+        msg_buffer_account_data.len(),
+        MessageBufferTestContext::DEFAULT_TARGET_SIZE as usize
+    );
+
+    let (bump, _version, _header_len, end_offsets) =
+        deserialize_msg_buffer_header(&msg_buffer_account_data);
+
+    assert_eq!(bump, msg_buffer_bump);
+    assert_eq!(end_offsets, [0u16; 255]);
+
+    println!("invalid buffer shrink");
```

Comment:
The comment says invalid but the code doesn't throw an error

Reply:
this was meant to just mark where in the test i was.  i will remove this